webpack/InsightsCloudSync/Components/ErrorEmptyState.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import {
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateVariant,
+  EmptyStateBody,
+} from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import { translate as __, sprintf } from 'foremanReact/common/I18n';
+import { selectError } from '../InsightsCloudSyncSelectors';
+
+export const ErrorEmptyState = () => {
+  const error = useSelector(selectError);
+  return (
+    <EmptyState variant={EmptyStateVariant.xl}>
+      <EmptyStateIcon icon={ExclamationCircleIcon} />
+      <EmptyStateBody>
+        <p>
+          {sprintf(
+            __(
+              'Something went wrong while getting the data, the server returned the following error: %s'
```

Comment:
Is it a network error? maybe we can just say "Network error, looks like something went wrong, please try to refresh the page"

Reply:
For example the error that was fixed in the table PR (undefined taxonamy_conditions). It's not a network error so refreshing won't fix it